docs/src/app/components/pages/components/avatars.jsx

```diff
@@ -38,7 +38,7 @@ export default class AvatarsPage extends React.Component {
           },
           {
             name: 'size',
-            type: 'number',
+            type: 'integer',
```

Comment:
i would want to stick the entire scala type system in there and make everything as specific as possible, so i'll leave that decision with you :D